content/influxdb/v2.0/influxdb-templates/monitor.md

```diff
@@ -0,0 +1,123 @@
+---
+title: Monitor InfluxDB OSS using a template
+description: >
+  Monitor your InfluxDB OSS instance using InfluxDB Cloud and
+  a pre-built InfluxDB template.
+menu:
+  influxdb_2_0:
+    parent: Monitor with templates
+    name: Monitor InfluxDB OSS
+weight: 102
+influxdb/v2.0/tags: [templates, monitor]
+related:
+  - /influxdb/v2.0/reference/cli/influx/apply/
+  - /influxdb/v2.0/reference/cli/influx/template/
+---
+
+Use [InfluxDB Cloud](/influxdb/cloud/), the [InfluxDB Open Source (OSS) Metrics template](https://github.com/influxdata/community-templates/tree/master/influxdb2_oss_metrics), and Telegraf to monitor one or more InfluxDB OSS instances.
+
+Do the following:
+
+1. [Review requirements](#review-requirements)
+2. [Install the InfluxDB OSS Monitoring template](#install-the-influxdb-oss-monitoring-template)
+3. [Set up InfluxDB OSS for monitoring](#set-up-influxdb-oss-for-monitoring)
+4. [Set up Telegraf](#set-up-telegraf)
+5. [View the Monitoring dashboard](#view-the-monitoring-dashboard)
+6. (Optional) [Alert when metrics stop reporting](#alert-when-metrics-stop-reporting)
+
+## Review requirements
+
+Before you begin, make sure you have access to the following:
+
+ - InfluxDB Cloud account ([sign up for free here](https://cloud2.influxdata.com/signup))
+ - Command line access to a machine [running InfluxDB OSS 2.x](/influxdb/v2.0/install/) and permissions to install Telegraf on this machine
+ - Internet connectivity from the machine running InfluxDB OSS 2.x and Telegraf to InfluxDB Cloud
+ - Sufficient resource availability to install the template. InfluxDB Cloud Free Plan accounts include [resource limits](/influxdb/cloud/account-management/pricing-plans/#resource-limits/influxdb/cloud/account-management/pricing-plans/#resource-limits).
+
+## Install the InfluxDB OSS Monitoring template
+
+The InfluxDB OSS Monitoring template includes a Telegraf configuration and a dashboard, making it easy to set up and monitor OSS.
+
+1. [Log into your InfluxDB Cloud account](https://cloud2.influxdata.com/), go to **Settings > Templates**, and enter the following template URL:
+
+    `https://raw.githubusercontent.com/influxdata/community-templates/master/influxdb2_oss_metrics/influxdb2_oss_metrics.yml`
+
+2. Click **Lookup Template**, and then click **Install Template**. InfluxDB Cloud imports the template, which includes the following resources:
+   - Dashboard `InfluxDB OSS Metrics`
+   - Telegraf configuration `scrape-infludb-oss-telegraf`
+   - Bucket `oss_metrics`
+   - Check `InfluxDB OSS Deadman`
+   - Labels `influxdb2` and `prometheus`
+
+## Set up InfluxDB OSS for monitoring
+
+By default, InfluxDB OSS 2.x has a `/metrics` endpoint available, which exports Prometheus-style system metrics.
+
+1. If you've changed the default settings to disable the `/metrics` endpoint, re-enable these settings.
```

Comment:
Should we assume folks know how to re-enable default settings for the `/metrics` endpoint (eg, if someone else disabled)? And/or what’s the easiest way to check enable/disable/default settings?